app/javascript/lib/Entities/AbstractImperialGame.js

```diff
@@ -8,14 +8,42 @@ export default class AbstractImperialGame extends Entity {
   }
 
   #rondel;
+  #availableFreeRondelSlotCount = 3;
+  #availablePaidRondelSlotCount = 3;
 
-  constructor(id) {
+  #nations;
+  #nationOrder;
+
+  constructor(id, nations, nationOrder) {
```

Comment:
`id` here is the game type, like `Imperial2030Game`, right? If so can we call it something other than `id`? maybe `baseGameId`?

Reply:
Originally I considered each game's id `abcde-4654-hijkl` to be part of the domain logic, and was leaving this id to populate that, but in your other comment you considered it as more of a web-concern, so Im not too sure how to utilize this 'id'

For now there hasn't been any need for it yet so it's just hanging around

I suppose my argument it for being a domain concern is that eventually users will need to refer to a specific game (ie. loading a past game, making moves across different games), and that's where it is referred to.
On the other hand I do agree that I dont ever envision it being used inside game logic so, is it really needed?

Still thinking about it, probably leaning towards your suggestion